src/services/EcoBridge/Xdai/XdaiBridge.selectors.ts

```diff
@@ -0,0 +1,128 @@
+import { createSelector } from '@reduxjs/toolkit'
+
+import { AppState } from '../../../state'
+import { BridgeTransactionStatus, BridgeTransactionSummary } from '../../../state/bridgeTransactions/types'
+import { normalizeInputValue } from '../../../utils'
+import { ArbitrumPendingReasons } from '../Arbitrum/ArbitrumBridge.types'
+import { XdaiBridgeList } from '../EcoBridge.types'
+import { xdaiBridgeTransactionAdapter } from './XdaiBridge.adapter'
+
+const createSelectBridgingDetails = (bridgeId: XdaiBridgeList) =>
+  createSelector(
+    [
+      (state: AppState) => state.ecoBridge[bridgeId].bridgingDetails,
+      (state: AppState) => state.ecoBridge[bridgeId].bridgingDetailsStatus,
+    ],
+    (details, loading) => {
+      return {
+        bridgeId,
+        details,
+        loading,
+        errorMessage: undefined,
+      }
+    }
+  )
+
+const createSelectOwnedTransactions = (bridgeId: XdaiBridgeList) => {
+  const transactionsSelector = createSelector(
+    [(state: AppState) => state.ecoBridge[bridgeId].transactions],
+    transactions => xdaiBridgeTransactionAdapter.getSelectors().selectAll(transactions)
+  )
+
+  return createSelector(
+    [transactionsSelector, (state: AppState, account: string | undefined) => account],
+    (transactions, account) => {
+      if (account) {
+        const normalizedAccount = account.toLowerCase()
+
+        return transactions.filter(tx => tx.sender.toLowerCase() === normalizedAccount)
+      }
+      return []
+    }
+  )
+}
```

Comment:
Can you check and make sure you are passing `account`  as second params where ever required else it always return an empty [] for transactions.

Reply:
Also suggest to create a` type Account = string | undefined;` and use that so it will show the second param is of type account/